pkg/arvo/app/notify.hoon

```diff
@@ -0,0 +1,449 @@
+::
+/-  *notify, resource, hark-store, post
+/+  default-agent, verb, dbug, group
+::
+|%
++$  card  card:agent:gall
+::
++$  provider-state  (map term provider-entry)
++$  provider-entry
+  $:  notify-endpoint=@t
+      binding-endpoint=@t
+      auth-token=@t
+      clients=(map ship (unit @t))
```

Comment:
Can you explain what the (unit @t) is? Having a hard time getting that from the code so far.

Reply:
A binding token, we set it when the call to register-binding returns, and it is used when we call remove-binding